lsp-mode.el

```diff
@@ -348,7 +348,7 @@ unless overridden by a more specific face association."
   :package-version '(lsp-mode . "6.1"))
 
 (defcustom lsp-client-packages
-  '(ccls lsp-clients lsp-clojure lsp-csharp lsp-css lsp-dart lsp-elm
+  '(ccls lsp-ada lsp-clients lsp-clojure lsp-csharp lsp-css lsp-dart lsp-dockerfile lsp-elm
```

Comment:
I think this PR should not add `lsp-dockerfile`, right?

Reply:
this is getting messy :)